components/Modal.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Modal = props => {
+  const [active, setActive] = React.useState(true);
```

Comment:
What's the benefit of using `React.useState` here over making `Modal` a component by using `extends React.Component` and using `this.state`?

Reply:
I was thinking that as we update existing components we can refactor to React hooks. The only real advantage I see in this case is that we can use a plain ole function and still manage state. (Not sure there is any actual advantage to that here either though)

Reply:
Interesting. I'll need to read up on React Hooks!